prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -196,8 +197,15 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
    * enrolment for checking validation instead of form state.
    */
   private getEnrolmentErrors(enrolment: Enrolment): ValidationErrors {
+    const isDeviceProvider = this.enrolmentService.enrolment.careSettings.some((careSetting) =>
+      careSetting.careSettingCode === CareSettingEnum.DEVICE_PROVIDER);
+    const isDeviceProviderIdentifier = this.enrolmentFormStateService.regulatoryFormState.deviceProviderIdentifier.value;
```

Comment:
You'll want this to be `hasDeviceProviderIdentifier`

Reply:
renamed